README.md

```diff
@@ -182,6 +182,38 @@ some_ros_package
 - When developing the firmware/Arduino parts of the software, we've made a complete arduino workspace in `src/firmware`. This way you don't need to worry about downloading the libraries yourself!
 - In order to use this, go to your Arduino IDE's Preferences dialog box and use `/your/path/to/Snowflake/src/firmware` as your sketchbook directory. Open arduino sketches in the workspace and they will work!
 
+### Udev Rules
+
+Udev provides a userspace solution for a dynamic /dev directory, with persistent device naming.
+
+- Rule Writing Guideline
+	- Creat file: /etc/udev/rules.d/10-local.rules (where xx represent the priority; Here 10 is higher than 50, the default)
```

Comment:
Spelling: `Create`

Reply:
We should probably make sure this is Snowbots specific, so we can overwrite it if needed (maybe `10-snowbots.rules`)

Reply:
changes made